src/sagemaker/jumpstart/types.py

```diff
@@ -98,6 +97,28 @@ def __repr__(self) -> str:
         }
         return f"{type(self).__name__} at {hex(id(self))}: {str(att_dict)}"
 
+    def to_json(self) -> Dict[str, Any]:
+        """Returns json representation of object."""
+        json_obj = {}
+        for att in self.__slots__:
+            if hasattr(self, att):
+                cur_val = getattr(self, att)
+                if issubclass(type(cur_val), JumpStartDataHolderType):
+                    json_obj[att] = cur_val.to_json()
+                elif isinstance(cur_val, list):
+                    json_obj[att] = []
+                    for obj in cur_val:
+                        if issubclass(type(obj), JumpStartDataHolderType):
+                            json_obj[att].append(obj.to_json())
+                        else:
+                            json_obj[att].append(obj)
+                else:
+                    json_obj[att] = cur_val
+        return json_obj
+
+
+from sagemaker.jumpstart.curated_hub.types import HubContentType  # noqa: E402, C0413
```

Comment:
Q- why isn't this import statement at the top?

Reply:
It was getting circular import pylint errors so without having to move the HubContentType back to jumpstart/types.py, this was a way to work around.

Reply:
wait this is blocking in my opinion, doing an import like this is an indication that the modules are not organized properly. Why can't we just put the hub content types in this module?